Cargo.toml

```diff
@@ -29,22 +29,22 @@ lru = "0.7.1"
 dashmap = "5.4"
 dyn-clone = "1.0"
 
-ckb-types = "=0.108.0"
-ckb-dao-utils = "=0.108.0"
-ckb-traits = "=0.108.0"
-ckb-jsonrpc-types = "=0.108.0"
-ckb-hash = "=0.108.0"
-ckb-resource = "=0.108.0"
-ckb-crypto = { version = "=0.108.0", features = ["secp"] }
-ckb-script = { version = "=0.108.0"}
+ckb-types = "0.111.0-rc8"
+ckb-dao-utils =  "0.111.0-rc8"
+ckb-traits =  "0.111.0-rc8"
+ckb-jsonrpc-types  =  "0.111.0-rc8"
+ckb-hash =  "0.111.0-rc8"
+ckb-resource =  "0.111.0-rc8"
+ckb-crypto = { version = "=0.111.0-rc8", features = ["secp"] }
+ckb-script = "0.111.0-rc8"
 bitflags = "1.3.2"
 sha3 = "0.10.1"
 enum-repr-derive = "0.2.0"
 
 # for feature test
 rand = { version = "0.7.3", optional = true }
-ckb-mock-tx-types = { version = "0.108.0", optional = true }
-ckb-chain-spec = { version = "0.108.0", optional = true }
+ckb-mock-tx-types = { git = "https://github.com/nervosnetwork/ckb-standalone-debugger", branch = "develop"}
```

Comment:
If we are going to publish ckb-sdk to crates.io, we would still need to publish this on crates.io.

Reply:
Yes.